### PR TITLE
Jomenxiao/add sqllogic test

### PR DESCRIPTION
**Pull request by jomenxiao**
add sqllogic test

### CODE REVIEW

File: stability-tester/serial_suite/sqllogictest.go [hunk 1]
```diff
@@ -0,0 +1,721 @@
...
+	"runtime/debug"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/ngaut/log"
+
```
Comment (cwen0) [inline stability-tester/serial_suite/sqllogictest.go:35]: Seems not well formatted.

File: stability-tester/serial_suite/sqllogictest.go [hunk 2]
```diff
@@ -0,0 +1,721 @@
...
+	if taskCount > len(fileNames) {
+		taskCount = len(fileNames)
+	}
+	if taskCount <= 0 {
+		return nil
+	}
+	taskChan := make(chan string, taskCount)
+	doneChan := make(chan struct{}, taskCount)
+	resultChan := make(chan *result, taskCount)
+	mdbs := createDatabases(taskCount, s.port, s.host, s.user, s.password, s.cfg.DBName)
```
Comment (cwen0) [inline stability-tester/serial_suite/sqllogictest.go:101]: one db is enough

Comment (jomenxiao) [reply stability-tester/serial_suite/sqllogictest.go:101]: nope. every case file create same table

File: stability-tester/serial_suite/sqllogictest.go (hunk 1 above)
Comment (jomenxiao) [reply stability-tester/serial_suite/sqllogictest.go:35]: 😿   formatted by vscode

File: stability-tester/serial_suite/sqllogictest.go (hunk 2 above)
Comment (cwen0) [reply stability-tester/serial_suite/sqllogictest.go:101]: but, `s.port 、 s.host 、cfg.DBName` are all the same.

**Comment by iamxy** [comment]
OMG !!!
@jomenxiao you have pushed the whole set of sql logic tests to this remote repo ?!!
the project becomes a huge size, every pulling will take a big long time ...

File: stability-tester/serial_suite/sqllogictest.go (hunk 2 above)
Comment (jomenxiao) [reply stability-tester/serial_suite/sqllogictest.go:101]: `cfg.DBName` not use.just create a connected.

**Comment by jomenxiao** [comment]
@iamxy 😢  remove it. and will download it by other way

**Comment by cwen0** [comment]
LGTM

**Review by cwen0 — APPROVED** [review]

**Pull request merged**